java/ql/src/experimental/semmle/code/java/Logging.qll

```diff
@@ -28,6 +27,23 @@ class LoggingCall extends MethodAccess {
       m.getReturnType() instanceof VoidType and
       this = m.getAReference()
     )
+    or
+    exists(RefType t, Method m | t.hasQualifiedName("android.util", "Log") |
+      (
+        m.hasName("d") or
```

Comment:
You could replace this with a set-literal :)
m.hasName(["d", "e", "i", "v", "w", "wtf"])

Reply:
Nice. Didn't know this was a thing :)